src/commands/time/_base.js

```diff
@@ -42,9 +41,14 @@ function captureNewTime(args, tpClient, done) {
                 if(!tpEntity) return '';
 
                 var task = { id: tpEntity.Id, name: tpEntity.Name, type: tpEntity.ResourceType.toLowerCase() };
-                var us  = tpEntity.UserStory ?
+                var us;
```

Comment:
I think the ternary inside an else should be refactored, e.g.:

var us = null;
if (tpEntity.ResourceType === 'UserStory') {
    us = { id: tpEntity.Id, name: tpEntity.Name };
} else if (tpEntity.UserStory) {
    us =  { id: tpEntity.UserStory.Id, name: tpEntity.UserStory.Name } 
}